lib/activerecord-bitemporal/callbacks.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module ActiveRecord::Bitemporal
+  module Callbacks
+    extend ActiveSupport::Concern
+
+    module Wrapper
+      def _create_record
+        perform_bitemporal_callbacks? ? run_callbacks(:bitemporal_create) { super } : super
+      end
+
+      def _update_record(*)
+        perform_bitemporal_callbacks? ? run_callbacks(:bitemporal_update) { super } : super
+      end
+
+      def destroy
+        perform_bitemporal_callbacks? ? run_callbacks(:bitemporal_destroy) { super } : super
+      end
+
+      def save_without_bitemporal_callbacks!(...)
+        with_bitemporal_option(ignore_bitemporal_callbacks: true) {
+          save!(...)
+        }
+      end
+
+      private
+
+      def perform_bitemporal_callbacks?
+        bitemporal_option[:ignore_bitemporal_callbacks] != true
+      end
+    end
+
+    included do
+      define_model_callbacks :bitemporal_create
+      define_model_callbacks :bitemporal_update
+      define_model_callbacks :bitemporal_destroy
+
+      prepend Wrapper
```

Comment:
Why do we need prepend here?

In the upstream, it seems that this is just be overridden.
https://github.com/rails/rails/blob/v7.0.4.2/activerecord/lib/active_record/callbacks.rb#L460-L462

Same for `ActiveRecord::Bitemporal::Persistence`.
https://github.com/kufu/activerecord-bitemporal/blob/05fcb5a3099e19e17fca1bc98f484c186be366a9/lib/activerecord-bitemporal/bitemporal.rb#L292-L298

Reply:
I modified not to prepend Wrapper module.